### DISCUSSION TITLE
how to set "own records" permissions

### DISCUSSION THREAD

**Discussion by thevegasmovement** [Budibase/budibase#7156] [Help] [answered]
I am building an app where there will be many end users but I only want them to have access to their own records. I had initially been waiting for  this to be pushed out in a future version and it's been like a year. Has this been created yet? If so, how would I do this? ty!

**Comment by shogunpurple** [comment, marked as answer]
@thevegasmovement please use the search before posting - this has been posted several times across both issues and discussions.

https://github.com/Budibase/budibase/issues/4789 is the main issue, but we don't have a timeline for it yet. We are hoping to get this one done before hopefully the end of the year.
